templates/templated_email/order/note/confirm_note.email

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+
+{% block subject %}
+  {% trans "New note" context "Note confirmation email subject" %}
```

Comment:
This title sounds too laconic, what about something like "New note has been added to your order"?